### ISSUE TITLE
ATLAddressBarViewController custom cell

### ISSUE THREAD

**Issue by mailyan** [layerhq/Atlas-iOS#860]
Hi, 

I want to customize ```ATLAddressBarViewController``` UITableView cells. I need to show participant avatar and customize the font when searching (see the image below). I think neither ```ATLAddressBarViewController``` nor ```ATLConversationViewController``` provide appropriate methods for this. What would be the best solution?

Thanks in advance.

![Chat]
(http://s28.postimg.org/nj13x39vx/chat.png)

**Comment by kmahal** [comment]
@mailyan Atlas currently does not support the customization of the `ATLAddressBarViewController`'s table view cells.  We use native `UITableViewCell`s:
```objc
- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
{
    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:ATLMParticpantCellIdentifier];
    id<ATLParticipant> participant = self.participants[indexPath.row];
    cell.textLabel.text = participant.fullName;
    cell.textLabel.font = ATLMediumFont(16);
    cell.textLabel.textColor = ATLBlueColor();
    return cell;
}
```
The font customization would be easier to implement with `UIAppearance` selectors like in other classes, but adding the avatar item will require registering a custom cell.

Feel free to submit a pull request for this feature, or fork the repository and change the implementation details for your use case.  I'll close out the issue unless you believe you'll be submitting a PR soon.  Thanks!

**Comment by mailyan** [comment]
@kmahal Thanks for a quick response.

I forked and changed the source code. I changed the cell type to conform  to `ATLParticipantPresenting` protocol. The default cell class is `ATLParticipantTableViewCell`. You can also set your custom cells conforming if they conform to `ATLParticipantPresenting` protocol via cellClass property. Cell row height, displays avatar and sort type can also be customized via appropriate properties. See pull request #864. Thanks.

**Comment by mailyan** [comment]
@kmahal Is there any update on this?

**Comment by kcoleman731** [comment]
Functionality was merged in #1188
